lib/svgo.test.js

```diff
@@ -64,6 +64,123 @@ test('allow to disable and customize plugins in preset', () => {
+    expect(data).toEqual(
+      `<svg viewBox="0 0 120 120">\n  <circle fill="red" cx="60" cy="60" r="50"/>\n</svg>\n`
+    );
+  });
+
+  test('should respect EOL set to CRLF', () => {
+    const svg = `
+      <?xml version="1.0" encoding="utf-8"?>
+      <svg viewBox="0 0 120 120">
+        <desc>
+          Not standard description
+        </desc>
+        <circle fill="#ff0000" cx="60" cy="60" r="50"/>
+      </svg>
+    `;
+    const { data } = optimize(svg, {
+      js2svg: { eol: 'crlf', pretty: true, indent: 2 },
+    });
+    // using toEqual because line endings matter in these tests
+    expect(data).toEqual(
+      `<svg viewBox="0 0 120 120">\r\n  <circle fill="red" cx="60" cy="60" r="50"/>\r\n</svg>\r\n`
+    );
+  });
+
+  test('should respect EOL set to line break', () => {
+    const svg = `
+      <?xml version="1.0" encoding="utf-8"?>
+      <svg viewBox="0 0 120 120">
+        <desc>
+          Not standard description
+        </desc>
+        <circle fill="#ff0000" cx="60" cy="60" r="50"/>
+      </svg>
+    `;
+    const { data } = optimize(svg, {
+      js2svg: { eol: '\r\n', pretty: true, indent: 2 },
+    });
+    // using toEqual because line endings matter in these tests
+    expect(data).toEqual(
+      `<svg viewBox="0 0 120 120">\r\n  <circle fill="red" cx="60" cy="60" r="50"/>\r\n</svg>\r\n`
+    );
+  });
+});
+
+describe('allow to configure final newline', () => {
+  test('should not add final newline when unset', () => {
+    const svg = `
+      <?xml version="1.0" encoding="utf-8"?>
+      <svg viewBox="0 0 120 120">
+        <desc>
+          Not standard description
+        </desc>
+        <circle fill="#ff0000" cx="60" cy="60" r="50"/>
+      </svg>
+    `;
+    const { data } = optimize(svg);
+    expect(data).toMatchInlineSnapshot(
+      `"<svg viewBox=\\"0 0 120 120\\"><circle fill=\\"red\\" cx=\\"60\\" cy=\\"60\\" r=\\"50\\"/></svg>"`
+    );
+  });
+
+  test('should add final newline when set', () => {
+    const svg = `
+      <?xml version="1.0" encoding="utf-8"?>
+      <svg viewBox="0 0 120 120">
+        <desc>
+          Not standard description
+        </desc>
+        <circle fill="#ff0000" cx="60" cy="60" r="50"/>
+      </svg>
+    `;
+    const { data } = optimize(svg, {
+      js2svg: { finalNewline: true },
+    });
+    expect(data).toMatchInlineSnapshot(`
+      "<svg viewBox=\\"0 0 120 120\\"><circle fill=\\"red\\" cx=\\"60\\" cy=\\"60\\" r=\\"50\\"/></svg>
+      "
+    `);
+  });
+
+  test('should not add extra newlines when using pretty: true', () => {
+    const svg = `
+      <?xml version="1.0" encoding="utf-8"?>
+      <svg viewBox="0 0 120 120">
+        <desc>
+          Not standard description
+        </desc>
+        <circle fill="#ff0000" cx="60" cy="60" r="50"/>
+      </svg>
+    `;
+    const { data } = optimize(svg, {
+      js2svg: { finalNewline: true, pretty: true, indent: 2 },
+    });
+    expect(data).toMatchInlineSnapshot(`
```

Comment:
I'd use toEqual in all cases here to highlight final newlines as well

Reply:
Addressed in 21bb65f.